activerecord/lib/active_record/fixtures.rb

```diff
@@ -589,7 +589,7 @@ def table_rows
 
           # interpolate the fixture label
           row.each do |key, value|
-            row[key] = label if value == "$LABEL"
```

Comment:
I believe we only need to do it if `value` is a string, so I think it is better to do:

```ruby
row[key] = label if value.is_a?(String) && value == "$LABEL"
```